notes-apps-ogechukwu-okoli.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+class Note:#base class
+    '''
+    It initializes new notes
+    '''
+    def __init__(self, note_id, content, created_at):
+        self.note_id = note_id # Notes_id
+        self.content = content # it contains the contents in the notes
+        self.created_at = created_at #store the created time stamp
+class TextNote(Note): #Child class
+    def __init__(self, note_id, content, created_at=None):
+        if created_at is None:
+            created_at = datetime.now() # time the note was created
+        Note.__init__(self, note_id, content, created_at)# initializing the parent class
```

Comment:
In child classes, consider using `super().__init__(...)` instead of directly calling the parent class’s initializer. It’s a bit cleaner and more future-proof.